.changeset/soft-sheep-glow.md

```diff
@@ -0,0 +1,18 @@
+---
+'@astrojs/alpinejs': patch
+'@astrojs/lit': patch
+'@astrojs/markdoc': patch
+'@astrojs/mdx': patch
+'@astrojs/node': patch
+'@astrojs/partytown': patch
+'@astrojs/preact': patch
+'@astrojs/react': patch
+'@astrojs/sitemap': patch
+'@astrojs/solid-js': patch
+'@astrojs/svelte': patch
+'@astrojs/tailwind': patch
+'@astrojs/vercel': patch
+'@astrojs/vue': patch
+---
+
+Update README
```

Comment:
```suggestion
Updates Integration README's
```

Reply:
I don't mind doing this, but since these are patches for the integration packages themselves, they'll show up in each of their individual changelogs as "README was updated", so I think this makes more sense?

Reply:
I will, however, fix the verb tense :joy: